python/commons/lab.py

```diff
@@ -0,0 +1,59 @@
+import sys
+import os
+import numpy as np
+
+import datajoint as dj
+
+schema = dj.schema('common_lab', locals())
+
+@schema
+class Paths(dj.Lookup):
+    definition = ...
+
+    def getLocalPath(self, path, local_os=None):
```

Comment:
Keeping with Python convention, let's call this `get_local_path`, using snake_case rather thank CamelCase. Although this makes the method name different from Matlab, it fits better with rest of the naming conventions.